GeorgieGirl24/adopt_dont_shop
# Story 2

Test
* Add users/show_spec.rb
* Check functionality of information shown to the page
MVC
* Create UsersController with show method
* Create users/show.html.erb with User's name, street address, city, state and zip code
* Create a model.rb file

Route
* Create GET route to users#show
